husky_components/column-navigation/main.js

```diff
@@ -409,14 +409,37 @@ define(function() {
             if (!!this.options.prefilledData) {
                 this.handleResponse(this.options.prefilledData, 0);
             } else {
-                this.load(this.options.url, 0);
+                this.load(this.getInitialUrl(), 0, function() {
+                    // when an error occured try once again with the error marked initial url
+                    this.initiallySelected = null;
+                    this.load(this.getInitialUrl(true), 0);
+                }.bind(this));
             }
         },
 
         /**
+         * Returns the url which is used to initially load the column-navigation
+         *
+         * @param {Boolean} errorOccured
+         * @returns {String} an url to load the column tree
+         */
+        getInitialUrl: function(errorOccured) {
```

Comment:
Can we introduce a `fallbackUrl` instead? Would be easier to understand, and is more flexible.